nat-lab/tests/helpers.py

```diff
@@ -366,3 +366,28 @@ async def setup_mesh_nodes(
         await connection_future
 
     return env
+
+
+def connectivity_stack(node1_stack: IPStack, node2_stack: IPStack) -> Optional[IPStack]:
+    """
+    Checks, through which paths (v4, v6 or both) the nodes can communicate with each other.
+    """
+    match (node1_stack, node2_stack):
```

Comment:
Python `match` statements are only supported on python 3.10 and later and I think we somewhat recently said on a different PR that we want to keep supporting python 3.9 for now, so this would need to be turned into an if-statement

Reply:
I probably missed that 3.9 support thingy, ok will downgrade to classics.

Reply:
It wasn't really an official thing, but I tried to introduce the same thing before, missing that it was only python 3.10+ and got a couple of comments on my PR that we're not ready to let go of python <3.10 yet

Reply:
Fixed